tests/test-cases/Group22-Docker-Apps/22-08-node.robot

```diff
@@ -48,7 +48,11 @@ Simple background node application
     Should Be Equal As Integers  ${rc}  0
     ${ip}=  Get IP Address of Container  node1
     
-    Wait Until Keyword Succeeds  10x  12s  Check node container  ${ip}
+    ${status}=  Get State Of Github Issue  8168
+    Run Keyword If  '${status}' == 'closed'  Fail  Test 22-08-node.robot needs to be updated now that Issue #8168 has been resolved
+
+    # once Issue #8168 is fixed, revert 60s to 12s
+    Wait Until Keyword Succeeds  10x  60s  Check node container  ${ip}
```

Comment:
from the experiment I did, it seems the current timeout 12s is enough for node:latest scenario. we don't need to change it.